hack/boilerplate/boilerplate.py

```diff
@@ -171,12 +172,23 @@ def get_files(extensions):
             outfiles.append(pathname)
     return outfiles
 
+def get_dates():
+    dates = ''
+    years = datetime.datetime.now().year
+
+    for year in range(2014, years+1):
+        dates += '|' + str(year)
+
+    dates = dates[1:]
+    return '(' + dates + ')'
```

Comment:
if you enjoy "python golf", you can combine a bunch of this together with the `str.join()` function and a list comprehension:
```python
return '(%s)' % '|'.join((str(y) for y in range(2014, years)))
```

(avoids the `dates[1:]` edge case, too)